srcpkgs/libXnvctrl/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'libXnvctrl'
+pkgname=libXnvctrl
+version=525.116.04
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+make_build_args="OUTPUTDIR=out"
+#make_install_args=""
+hostmakedepends="inetutils base-devel"
+makedepends="jansson-devel gtk+3-devel libXv-devel libvdpau-devel libXext-devel libXxf86vm-devel"
+short_desc="Nvidia hardware monitoring library"
+maintainer="Kira Taylor Patton <roundduckkira@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/NVIDIA/nvidia-settings"
+#changelog=""
+distfiles="https://github.com/NVIDIA/nvidia-settings/archive/refs/tags/${version}.tar.gz"
+checksum=32db97759c2a58fea86a63a69a423f1bf65198496cb2ac2279f4210b50097358
+
+do_install() {
+	vcopy src/out/libXNVCtrl.* /usr/lib/
+}
+
+libXnvctrl-devel_package() {
+	short_desc+=" - development files"
+	vmkdir /usr/include/NVCtrl 755
+	vcopy src/libXNVCtrl/*.h /usr/include/NVCtrl/
```

Comment:
this is missing a pkg_install function, see `zlib-devel`.

Reply:
ah crap, whoops I forgot that bit